SU2_CFD/src/solver_structure.cpp

```diff
@@ -2250,53 +2254,53 @@ void CSolver::Read_SU2_Restart_Metadata(CGeometry *geometry, CConfig *config, st
 
     /*--- Compute (negative) displacements and grab the metadata. ---*/
 
-    fseek(fhw,-(sizeof(int) + 5*sizeof(passivedouble)), SEEK_END);
+		fseek(fhw,-(sizeof(int) + 5*sizeof(passivedouble)), SEEK_END);
```

Comment:
Got catch! In the future, it should be more easy to add extra metadata... maybe the 8 should be a variable that you can change only in one place